state/upgrades.go

```diff
@@ -2942,6 +2939,9 @@ func ReplaceNeverSetWithUnset(pool *StatePool) (err error) {
 				Update: update,
 			})
 		}
+		if err := iter.Close(); err != nil {
+			return errors.Trace(err)
+		}
 
 		return errors.Trace(st.db().RunTransaction(ops))
```

Comment:
I am concerned that this step still runs all ops in a single transaction. On Prodstack there were 130k documents that needed updating. And that is a very hard txn to run, and to progress.

Reply:
I think my fears are well founded. Prodstack is dying essentially every 1 hr, and that is likely because txn pruner is failing to start, and *that* is likely triggered by the fact that we have a txn that is touching 130k records.